patches/xrootd.patch

```diff
@@ -1,85 +1,188 @@
-diff -u -r xrootd-orig/xrootd.spec xrootd-patched/xrootd.spec
---- xrootd-orig/xrootd.spec	2018-01-29 16:28:46.000000000 +0100
-+++ xrootd-patched/xrootd.spec	2018-03-28 08:55:47.389391917 +0200
-@@ -64,11 +64,6 @@
- BuildRequires: ncurses-devel
- 
- BuildRequires: python2-devel
--%if %{?fedora}%{!?fedora:0} >= 13
--BuildRequires: python3-devel
--%else
--BuildRequires: python34-devel
+--- /xrootd-orig/xrootd.spec	2018-05-03 08:20:43.000000000 +0200
++++ /xrootd-patched/xrootd.spec	2018-06-25 16:09:32.890745565 +0200
+@@ -37,18 +37,22 @@
+ BuildRequires:	systemd
+ %endif
+ BuildRequires:	python2-devel
+-%if %{?fedora}%{!?fedora:0} || %{?rhel}%{!?rhel:0} >= 8
+-BuildRequires:	python3-devel
 -%endif
- 
- BuildRequires: openssl-devel
- 
-@@ -276,19 +271,6 @@
- %description -n python2-xrootd
- Python 2 bindings for XRootD
- 
--#-------------------------------------------------------------------------------
--# python3
--#-------------------------------------------------------------------------------
--%package -n python3-%{name}
--Summary:       Python 3 bindings for XRootD
--Group:         Development/Libraries
--%if %{?fedora}%{!?fedora:0} >= 13
--%{?python_provide:%python_provide python3-%{name}}
+-%if %{?rhel}%{!?rhel:0} == 7
+-BuildRequires:	python34-devel
 -%endif
--Requires:      %{name}-client-libs%{?_isa} = %{epoch}:%{version}-%{release}
--
--%description -n python3-xrootd
--Python 3 bindings for XRootD
- 
- #-------------------------------------------------------------------------------
- # doc
-@@ -331,17 +313,6 @@
- Ceph back-end plug-in for XRootD.
- %endif
+-%if %{?fedora}%{!?fedora:0} || %{?rhel}%{!?rhel:0} >= 7
+-BuildRequires:	python2-sphinx
+-%endif
+-%if %{?rhel}%{!?rhel:0} == 6
+-BuildRequires:	python-sphinx10
+-%endif
++
++
++
```

Comment:
That is strange. Why do you have so many empty lines ?

Reply:
I removed everything was breaking, Python3, python34, python-sphinx10, python2-sphinx and all the documentation creation that was failing, and left the lines empty... if it is better I can remove the empty lines?